remo/profiles/tasks.py

```diff
@@ -83,6 +83,7 @@ def reset_rotm_nominees():
         nominees = UserProfile.objects.filter(is_rotm_nominee=True)
         for nominee in nominees:
             nominee.is_rotm_nominee = False
+            nominee.rotm_nominated_by = None
```

Comment:
@safwanrahman how do you feel to take this a step further by adding a different db table for the ROTM functionality?

This would be something like:
```python
class RotmNominee(models.Model):
    nominee = models.ForeignKey(User)
    nominated_by = models.ForeignKey(User)
    created_on = models.DateTimeField(auto_now_add=True)
    updated_on = models.DateTiemField(auto_now=True)
```

This way we won't have to reset the nominations on each month - we can simply create a new entry, and we can also have a modelform for the ROTM nominees.

Since this improvement was not part of the bug you are working on and came up through your work, we can always tackle it in another bug/PR. Thoughts?